arm-sql/2015-05-01-preview/swagger/serverKeys.json

```diff
@@ -0,0 +1,391 @@
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServerKeyListResult": {
+      "description": "A list of server keys.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ServerKey"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ServerKeyProperties": {
+      "description": "Properties for a server key execution.",
+      "type": "object",
+      "properties": {
+        "subregion": {
+          "description": "Subregion of the server key.",
+          "type": "string",
+          "readOnly": true
+        },
+        "serverKeyType": {
+          "description": "The server key type like 'ServiceManaged', 'AzureKeyVault'.",
+          "enum": [
+            "ServiceManaged",
+            "AzureKeyVault"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ServerKeyType",
+            "modelAsString": true
+          }
+        },
+        "uri": {
+          "description": "The URI of the server key.",
+          "type": "string"
+        },
+        "thumbprint": {
+          "description": "Thumbprint of the server key.",
+          "type": "string"
+        },
+        "creationDate": {
+          "format": "date-time",
+          "description": "The server key creation date.",
+          "type": "string"
+        }
+      }
+    },
+    "Resource": {
+      "description": "ARM resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ProxyResource": {
+      "description": "ARM proxy resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {}
+    },
+    "ServerKey": {
+      "description": "A server key.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "description": "Kind of encryption protector. This is metadata used for the Azure portal experience.",
+          "enum": [
+            "None",
+            "azurekeyvault",
+            "servicemanaged"
+          ],
+          "type": "string"
+        },
+        "location": {
```

Comment:
Just to confirm, by adding location  you dont intend to make this a tracked resource.